app/src/main/java/org/dslul/openboard/inputmethod/latin/settings/CorrectionSettingsFragment.java

```diff
@@ -47,9 +47,15 @@ public final class CorrectionSettingsFragment extends SubScreenFragment
     implements SharedPreferences.OnSharedPreferenceChangeListener,
         PermissionsManager.PermissionsResultCallback {
 
-    private static final boolean DBG_USE_INTERNAL_PERSONAL_DICTIONARY_SETTINGS = false;
-    private static final boolean USE_INTERNAL_PERSONAL_DICTIONARY_SETTINGS =
-            DBG_USE_INTERNAL_PERSONAL_DICTIONARY_SETTINGS;
+    private static final boolean OPENBOARD_PERSONAL_DICTIONARY_SETTINGS = true;
+    private boolean SYSTEM_PERSONAL_DICTIONARY_SETTINGS() {
```

Comment:
Using _static_ style for function names is not technically wrong, but very misleading. Function names should use camelCase.

Reply:
I'll correct it right away.
(Thanks for the camelCase term; I didn't know it)

Reply:
Done with commit da2d29f.